infra/config/example.ts

```diff
@@ -18,6 +18,8 @@ export const config: EnvConfig = {
     FITBIT_CLIENT_SECRET: "FITBIT_CLIENT_SECRET",
     GARMIN_CONSUMER_KEY: "GARMIN_CONSUMER_KEY",
     GARMIN_CONSUMER_SECRET: "GARMIN_CONSUMER_SECRET",
+    GOOGLE_CONSUMER_KEY: "GOOGLE_CONSUMER_KEY",
+    GOOGLE_CONSUMER_SECRET: "GOOGLE_CONSUMER_SECRET",
```

Comment:
Not clientId and ClientSecret? This is oauth2.

Also, don't forget to update the the readme and the internal repo OSS config with these new secrets.